apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.csproj

```diff
@@ -13,10 +13,9 @@
     <PackageReference Include="Google.Api.Gax.Grpc.GrpcCore" Version="[3.2.0, 4.0.0)" />
     <PackageReference Include="Google.Cloud.Logging.V2" Version="[3.0.0, 4.0.0)" />
     <PackageReference Include="Google.Cloud.Trace.V1" Version="[2.0.0, 3.0.0)" />
-    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.1.1" />
     <PackageReference Include="Microsoft.DotNet.Analyzers.Compatibility" Version="0.2.12-alpha" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.Extensions.Http" Version="2.1.1" />
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="All" />
+	<PackageReference Include="Microsoft.Extensions.Http" Version="2.1.1" />
```

Comment:
More odd indentation... looks like it's a tab instead of spaces... worth checking settings.